client/src/ui/ClassView.jsx

```diff
@@ -35,8 +37,9 @@ export class ClassView extends Component {
     super(props);
 
     // grabs class number and subject from the GET parameters
-    const number = this.props.match.params.number;
-    const subject = this.props.match.params.subject.toLowerCase();
+    console.log(props);
```

Comment:
🤔

Reply:
Ah, Meteor did some odd props-merging magic that no longer existed. Took a print or two to figure out what was happening.